kotlin/injivcrenderer/build.gradle.kts

```diff
@@ -0,0 +1,79 @@
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.jetbrains.kotlin.android)
+    `maven-publish`
+}
+
+android {
+    namespace = "io.mosip.injivcrenderer"
+    compileSdk = 34
+
+    defaultConfig {
+        minSdk = 23
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles("consumer-rules.pro")
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+                "proguard-rules.pro"
+            )
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_17
```

Comment:
did we try with java21?

Reply:
Yes,tried with Java21. when Integrating in Inji, it gives error. So only used Java17